packages/react-scripts/template/README.md

```diff
@@ -675,6 +675,14 @@ The watcher includes an interactive command-line interface with the ability to r
 
 ![Jest watch mode](http://facebook.github.io/jest/img/blog/15-watch.gif)
 
+>**A note for Mac OSX users**
+
+>If you start `npm test` and the execution hangs there might be a problem with your watchman installation as discovered in #713 and #1767.
+
+> To solve the issues first make sure you run the latest verision: `brew install watchman` (4.7.0).
```

Comment:
typo, should be "version".
Let's not hardcode specific version but say "make sure you run at least watchman 4.7.0 or newer".